src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/DataprocConfig.scala

```diff
@@ -3,7 +3,7 @@ package org.broadinstitute.dsde.workbench.leonardo.config
 case class DataprocConfig(applicationName: String,
                           serviceAccount: String,
                           dataprocDefaultRegion: String,
-                          leoGoogleBucket: String,
+                          leoGoogleProject: String,
```

Comment:
Thank you! This bothered me as well.

Reply:
Side note: it might be nice to use our rich value objects in config classes as well.